samples/04.0-file-transfer/README.md

```diff
@@ -34,45 +35,45 @@ asset.
 // in FileTransferExtension.java
 @Override
 public void initialize(ServiceExtensionContext context){
-    // ...
-    var policy = createPolicy();
-    policyStore.save(policy);
-    
-    registerDataEntries(context);
-    registerContractDefinition(policy.getUid());
-    // ...
-}
+        // ...
+        var policy=createPolicy();
+        policyStore.save(policy);
 
-//...
+        registerDataEntries(context);
+        registerContractDefinition(policy.getUid());
+        // ...
+        }
 
-private void registerDataEntries(ServiceExtensionContext context) {
-    var assetPathSetting = context.getSetting(EDC_ASSET_PATH, "/tmp/provider/test-document.txt");
-    var assetPath = Path.of(assetPathSetting);
-    
-    var dataAddress = DataAddress.Builder.newInstance()
-            .property("type", "File")
-            .property("path", assetPath.getParent().toString())
-            .property("filename", assetPath.getFileName().toString())
-            .build();
-    
-    var assetId = "test-document";
-    var asset = Asset.Builder.newInstance().id(assetId).build();
-    
-    loader.accept(asset, dataAddress);
-}
+//...
 
-private void registerContractDefinition(String uid) {
-    var contractDefinition = ContractDefinition.Builder.newInstance()
-            .id("1")
-            .accessPolicyId(uid)
-            .contractPolicyId(uid)
-            .selectorExpression(AssetSelectorExpression.Builder.newInstance()
-                    .whenEquals(Asset.PROPERTY_ID, "test-document")
-                    .build())
-            .build();
-    
-    contractStore.save(contractDefinition);
-}
+private void registerDataEntries(ServiceExtensionContext context){
+        var assetPathSetting=context.getSetting(EDC_ASSET_PATH,"/tmp/provider/test-document.txt");
+        var assetPath=Path.of(assetPathSetting);
+
+        var dataAddress=DataAddress.Builder.newInstance()
+        .property("type","File")
+        .property("path",assetPath.getParent().toString())
+        .property("filename",assetPath.getFileName().toString())
+        .build();
+
+        var assetId="test-document";
+        var asset=Asset.Builder.newInstance().id(assetId).build();
+
+        loader.accept(asset,dataAddress);
+        }
+
+private void registerContractDefinition(String uid){
+        var contractDefinition=ContractDefinition.Builder.newInstance()
+        .id("1")
+        .accessPolicyId(uid)
+        .contractPolicyId(uid)
+        .selectorExpression(AssetSelectorExpression.Builder.newInstance()
+        .whenEquals(Asset.PROPERTY_ID,"test-document")
+        .build())
+        .build();
+
+        contractStore.save(contractDefinition);
+        }
```

Comment:
Just a minor thing, but can you restore the indentation here?

Reply:
:wave:  sure i guess it was the IDE that did the reformat :)